File: derekbanas/js/00_prime.js
function isPrime(n) {
    if (n === 1 || n === 2 || n % 2 == 2) return false;

    for (var i = 3, j = parseInt(Math.sqrt(n)); i <= j; i ++ ) {
        if (n % i == 0) {
            return false;
        }
    }
    return true;
}


console.log(isPrime(11))
console.log(isPrime(81))

File: derekbanas/python/00_prime.py
# http://www.algolist.net/Algorithms/Number_theoretic/Primality_test_naive


from math import sqrt

def isprime(n):
    if n == 1 or n == 2 or n % 2 == 0:
        return False
    for i in range( 3, int(sqrt(n)) + 1, 1):
        if n % i == 0:
            return False

    return True




print isprime(11)
print isprime(1000)